bin/post_process_sce.R

```diff
@@ -246,8 +246,8 @@ try({
 
 # Export --------------
 
-# write out original SCE with additional filtering column
-readr::write_rds(sce, opt$input_sce_file, compress = "gz")
+# write out _original_ filtered SCE with additional filtering column
+readr::write_rds(sce, opt$input_filtered_sce_file, compress = "gz")
```

Comment:
Good argument to maybe rename this just `filtered_sce_file` 
```suggestion
readr::write_rds(sce, opt$input_filtered_sce_file, compress = "gz")
```